### DISCUSSION TITLE
ESP32-WROOM-32E and KSZ8863RLL

### DISCUSSION THREAD

**Discussion by CaCtus491** [forkineye/ESPixelStick#691] [General]
Just checking in to see if anyone has attempted to use ESPixelStick with the KSZ8863RLL three port switch chip? I have successfully used the ESP32 IDF to create a (unrelated) project on custom hardware that works with the LAN8720, however new to me is the KSZ8863RLL and ESPixelStick.

Espressif have a sample project for the KSZ8863RLL (https://github.com/espressif/esp-eth-drivers/blob/master/ksz8863/README.md), and the hardware design borrows heavily from their reference design.

Once I've validated the hardware using the project above, I intend to fold it into ESPixelStick through a new board / platform config.

Any pointers for someone new to ESPixelStick for this sort of integration are very much welcome.

Looking to get this operational over the next two weeks (once the boards arrive).

**Comment by MartinMueller2003** [comment]
Please connect with me on Discord and I will walk you through the process.

**Comment by MartinMueller2003** [comment]
I am still working on this in the background.
